host-controller/src/main/java/org/jboss/as/domain/controller/logging/DomainControllerLogger.java

```diff
@@ -802,4 +802,7 @@
 
     @Message(id = 97, value = "Cannot explode a subdeployment of an unexploded deployment")
     OperationFailedException cannotExplodeSubDeploymentOfUnexplodedDeployment();
+
+    @Message(id = 98, value = "You are trying to redeploy rutimes not affected by the overlay")
```

Comment:
Actually better is ""You are trying to redeploy deployments not affected by the overlay", as "runtimes" has no clear meaning. Do you have the list of invalid names? If so, or if it's easy to get it, it's better to include the list in the message.